engine/lib/elgglib.php

```diff
@@ -1924,6 +1924,8 @@ function _elgg_engine_boot() {
 function _elgg_init() {
 	global $CONFIG;
 
+	elgg_register_action('delete');
```

Comment:
I wonder if this should be `entity/delete`? After all we could also have `relationship/delete`, `annotation/delete`, etc.